tests/Elm/Parser/ExpressionTests.elm

```diff
@@ -590,6 +601,116 @@ all =
                                 ]
                             )
                         )
+        , test "negated expression after = in record" <|
+            \() ->
+                "{a=-x}"
+                    |> expectAst
+                        (Node { start = { row = 1, column = 1 }, end = { row = 1, column = 7 } }
+                            (RecordExpr
+                                [ Node { start = { row = 1, column = 2 }, end = { row = 1, column = 6 } }
+                                    ( Node { start = { row = 1, column = 2 }, end = { row = 1, column = 3 } } "a"
+                                    , Node { start = { row = 1, column = 4 }, end = { row = 1, column = 6 } }
+                                        (Negation
+                                            (Node { start = { row = 1, column = 5 }, end = { row = 1, column = 6 } } (FunctionOrValue [] "x"))
+                                        )
+                                    )
+                                ]
+                            )
+                        )
+        , test "negated expression after ==" <|
+            \() ->
+                "a==-x"
```

Comment:
in theory this one should not parse if we want to be that strict
```
-- UNKNOWN OPERATOR ------------------------------------------------------- REPL

I do not recognize the (==-) operator.

1|   a==-x
      ^^^
Is there an `import` and `exposing` entry for it? Maybe you want (==) or (-)
instead?
```

Reply:
Oops, right.

And I agree that the current system is not enough for `in`.